src/mvs/meshing.cc

```diff
@@ -849,7 +849,7 @@ PlyMesh DelaunayMeshing(const DelaunayMeshingOptions& options,
     }
 
     // Pop the next results from the queue.
-    const auto result = result_queue.Pop();
```

Comment:
`auto&& result = result_queue->Pop();`
The `result` is a right reference of the RValue of Pop. 
I suppose the `&&` should be kept, otherwise a default constructor and move assignment will be called.

Reply:
RVO should avoid any copies here without the &&? Even if it cannot directly place the object into the caller stack, it will do a move? Might be easy enough to verify by deleting the copy constructor/operator of the Data container.

Reply:
Yes, you are right. I have test the case. RVO works fine.